source/educators/how-tos/course_development/manage_video_transcripts.rst

```diff
@@ -35,25 +35,37 @@ transcript with that name. To avoid this problem, a best practice is to name
 additional transcript files by appending the transcript's language to the video
 file name.
 
-For example, you have two videos, named video1.mp4 and video2.mp4, and their
-original transcripts are named video1.srt and video2.srt. When you add Hindi
-transcripts for these videos, you name them video1_hi.srt and video2_hi.srt.
-When you add Spanish transcripts for the videos, you name them video1_es.srt
-and video2_es.srt.
+For example, you have two videos, named ``video1.mp4`` and ``video2.mp4``, and their
+original transcripts are named ``video1.srt`` and ``video2.srt``. When you add Hindi
+transcripts for these videos, you name them ``video1_hi.srt`` and ``video2_hi.srt``.
+When you add Spanish transcripts for the videos, you name them ``video1_es.srt``
+and ``video2_es.srt``.
 
 To add a transcript to a video component, follow these steps.
 
-#. After you obtain the .srt files, open the video component for the video.
+#. After you obtain the ``.srt`` files, open the video component for the video.
 
 #. Scroll down to the **Transcript** section, and then select **Add a
    transcript**.
 
+   .. image:: /_images/educator_how_tos/add_a_transcript.png
+    :alt: A screenshot of the video editor with the Add a Transcript section highlighted.
```

Comment:
This is a matter of taste; some people treat "a screenshot of" as equally as unnecessary in alt as "an image of"; other people think it's adding actual information. I usually lean toward the idea that in docs, "A screenshot of" is obvious from context.  That being said, consistency across all the docs is most important.
```suggestion
    :alt: The video editor with the Add a Transcript section highlighted.
```